packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -286,6 +286,7 @@ async function nodeImport(
       importer,
       // Non-external modules can import ESM-only modules, but only outside
       // of test runs, because we use Node `require` in Jest to avoid segfault.
+      // @ts-expect-error
```

Comment:
do we still need the next line with the special logic for jest?

Reply:
I was thinking to remove it as well. But I guess maybe what would affects users that testing with Jest on userland?

Reply:
Yes, better to keep it a bit more.